### ISSUE TITLE
Allow other languages?

### ISSUE THREAD

**Issue by brettz9** [vokeio/language-javascript-plus#12]
While [language-babel](https://github.com/gandm/language-babel#javascript-tagged-template-literal-grammar-extensions) already appears to do this, I'm wondering if you might similarly support a simple option to add other languages (e.g., a space-separated input box in settings) (I'm not interested in all of the other Babel features of that package).

In looking at https://github.com/vokeio/language-javascript-plus/blob/master/index.js , it seems it might not be too complicated to simply concat onto the `languages` array there, using the user's extra languages?

**Comment by brettz9** [comment]
Btw, you might want to build the `languages` array outside of the loop for somewhat more succinct and performant code.

**Comment by xeaone** [comment]
Yes more languages would be great! That was also my feeling about Babel's. I just have not needed any other languages so I have not had the drive or time.

And yes that is a great idea for a performance boost. 

As long as we continue with the same pattern for language detection using the comments these changes would be great. If you could make a pull request with the changes that you want I would gladly look it over and merge it.

**Comment by brettz9** [comment]
My plate is really full these days, so I don't expect to be able to get to it, at least for a good while. But might in the future if I get the chance.

**Comment by sabahmax-inc** [comment]
Hi! I’ve opened PR #15 to address this. It adds a simple settings option for extra languages as a space-separated list and concatenates those onto the existing `languages` array, without adding the other Babel features.
